### ISSUE TITLE
Re-Implementation in GluonCV

### ISSUE THREAD

**Issue by hetong007** [Microsoft/human-pose-estimation.pytorch#96]
Hi,

We have re-implemented the estimation model in [GluonCV](https://gluon-cv.mxnet.io/). We also achieve slightly higher AP by using pre-trained ResNet with higher accuracy: https://gluon-cv.mxnet.io/model_zoo/pose.html 

Besides, you mentioned in https://github.com/Microsoft/human-pose-estimation.pytorch/issues/2 that the tracking code is implemented with MXNet. Do you think there's a chance that you will release it? GluonCV is also based on MXNet. With your tracking implementation as a starting point, we may provide a smooth and unified pipeline of pose tracking in GluonCV.

Best,
Tong

**Comment by lxy5513** [comment]
Cool

**Comment by leoxiaobin** [comment]
@hetong007 Thanks for you excellent work. 

I am busy for another project, so when I am free I will consider to release the  tracking part. By the way, our original implementation is also based on gluon.  I think I am among the first users of gluon. :)

**Comment by hetong007** [comment]
That's fantastic to hear! Would you also share some thoughts on gluon? Like are you still using it, or what turns you to pytorch, etc. It is important to hear from our customers so that we can put efforts to improve on the most important parts.

**Comment by Odaimoko** [comment]
> @hetong007 Thanks for you excellent work.
> 
> I am busy for another project, so when I am free I will consider to release the tracking part. By the way, our original implementation is also based on gluon. I think I am among the first users of gluon. :)

Could you give a rough date when you will release the tracking part?
